_posts/Summary Script, Ep. 1 Open Source Directions: Spyder

```diff
@@ -0,0 +1,106 @@
+---
```

Comment:
This filename needs to be changed to match the filename of the post, ie something like `YYYY-MM-DD-title-without-spaces.md`